### DISCUSSION TITLE
Error Commands out of sync; you can't run this command now

### DISCUSSION THREAD

**Discussion by raibann** [sqlalchemy/sqlalchemy#10991] [Usage Questions]
`
 statement = text(f"""
                         select op.customfieldoptionid as id, op.optionvalue as value, op.isselected 
                         from swcustomfieldoptions op where op.customfieldid = :customfieldid;#test
                         """)
        # Response data
`
when i use `;#test` in statement it throw error " _mysql_connector.MySQLInterfaceError: Commands out of sync; you can't run this command now" 
*use in python

**Comment by CaselIT** [comment]
Hi,

The driver likely doesn't support multiple statements in the same execute call

**Reply by gordthompson** [reply]
I get that error with MySQL Connector/Python 8.3.0, but not with PyMySQL.
